midi-to-part-mp3s

```diff
@@ -20,17 +20,25 @@ class Part:
         return self.midi_filepath.replace(".midi", ".mp3")
 
 def track_names(midi_data):
-  return list(map(lambda track: track.name, midi_dat.tracks))
+  return list(map(lambda track: track.name, midi_data.tracks))
 
 def instrument_number_for_part(part):
+  if (args.instrument):
+     mapping = {
+            "soprano": args.instrument,
+            "alt": args.instrument,
+            "tenor": args.instrument,
+            "bass": args.instrument
+            }
+  else:
     # instrument choices based off advice given here: http://www3.cpdl.org/wiki/index.php/User:Robert_Urmann
     mapping = {
             "soprano": 76, # pan flute
             "alt": 75, # recorder
             "tenor": 49, # string ensemble 1
             "bass": 50 # string ensemble 2
             }
-    return mapping[part]
+  return mapping[part]
```

Comment:
Useful feature, do like.

I see a way to make this simpler. Instead of the mapping lines you added, you could modify the original code to read something like (untested code)`

```python
if (args.instrument):
   return args.instrument
else:
   ... # original code
```

Reply:
Ah you're right, I did not see that you only return the instrument number and not the complete mapping. I will fix that.